src/action.c

```diff
@@ -340,6 +344,14 @@ action_arg_from_xml_node(struct action *action, const char *nodename, const char
 			goto cleanup;
 		}
 		break;
+	case ACTION_TYPE_TOGGLE_DECORATIONS:
+	case ACTION_TYPE_UNDECORATE:
+		if (!strcasecmp(argument, "keepBorder")) {
+			action_arg_add_bool(action, argument,
+				parse_bool(content, rc.ssd_keep_border));
```

Comment:
I'm not sure that this will work as intended; `rc.ssd_keep_border` might not be properly set, depending on the order of lines in `rc.xml` and the state of the parser. Some testing will be required.

Reply:
Agree, this should likely just add the argument. `action_get_bool()` in `actions_run()` already falls back to `rc.ssd_keep_border`.

Reply:
I wasn't sure what `action_arg_from_xml_node()` is used for. If you say it is fine to use `true` here I can change that.